rdflib/graph.py

```diff
@@ -400,6 +409,17 @@ def addN(self, quads):
             and _assertnode(s, p, o)
         )
 
+    def addNAll(self, quads):
```

Comment:
the existing `addN` function already accepts a list of multiple quads. Adding `addNAll` is redundant.